app/views/foods/_food.html.erb

```diff
@@ -0,0 +1,15 @@
+
+<div id="<%= dom_id food %>" class="border-t border-gray-200">
+  <dl class="bg-gray-50 px-4 py-5 sm:grid sm:grid-cols-4 sm:gap-4 sm:px-6">
+    
+      <dt class="text-sm font-medium text-gray-500"><%= food.name %></dt>
+      <dd class="mt-1 text-sm text-gray-500"><%= food.measurement_unit %></dd>
+      <dd class="mt-1 text-sm text-gray-500"><%= food.price %></dd>
+      <dd class="mt-1 text-sm text-gray-500">
+        <% if current_user.id == food.user_id %>
+        <%= button_to "Destroy this food", food, method: :delete, class: "ml-5 bg-transparent hover:bg-blue-500 text-blue-700 font-semibold hover:text-white py-2 rounded-md shadow-sm text-sm leading-4 px-3 border border-blue-500 hover:border-transparent rounded focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500" %>
+        <% end %>
```

Comment:
Kindly update this code to use the `cancancan` authorization gem to control access to different resources

Reply:
Ok  Bro.